.storybook/preview.js

```diff
@@ -17,7 +17,7 @@ export const parameters = {
   docs: {
     source: {
       excludeDecorators: true,
-      type: 'source',
+      type: 'code',
```

Comment:
don't check this in

Reply:
Once this is removed we can review stuff that's owned by `v-build` (only this file).